kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/GameSaleRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.time.LocalDate;
+
+public class GameSaleRepository implements SalesRepository {
```

Comment:
I have some doubts about name of this repository: GameSaleRepository. It suggests that it allows to only sale games, but argument to method is just Product.

Let's think about possible implementation of SalesRepository:
- information about sales can be stored inside SQL Database - then it can be names SQLSaleRepository
- information about sales can be stored in memory - (InMemorySaleRepository)
- information about sales can be stored in file - (InFileSaleRepository)